README.md

```diff
@@ -8,61 +8,79 @@ Head over to the [marketplace](https://www.contentful.com/marketplace/) and foll
+  - **The Contentful ecosystem team will update the value of `APP_DEFINITION_ID` to a shared, official space. Please do not update it in subsequent changes/pull requests.**
+
+- Add the new app to the repo's dependabot configuration
+
+- Add the new app to the release-please-config.json file so that the release workflow will include the new app. The release workflow encapsulates versioning as well as the creation and maintenance of a changelog
 
 ### Committing and updating your repo
 
-When you are ready to submit your app to the [marketplace apps repo](https://github.com/contentful/marketplace-partner-apps), clone the repo locally, create a new branch with a descriptive name for the app/changes you are introducing, and create a pull-request against the `main` branch. An ecosystems integrations will review your PR soon and let you know if any changes or clarifications are needed. To ensure the fastest approval time on your PR, make sure you:
+When you are ready to submit your app to the [marketplace apps repo](https://github.com/contentful/marketplace-partner-apps), clone the repo locally, create a new branch with a descriptive name for the app/changes you are introducing, and create a pull-request against the `main` branch. An ecosystem team member will review your PR soon and let you know if any changes or clarifications are needed. To ensure the fastest approval time on your PR, make sure you:
 
 - PR titles must confirm to the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) standard. (A Github Action will emit a failing status check if your PR title does not match)
 - **Commit messages are public!** We highly recommend following standards for [good commit messages](https://github.com/googleapis/release-please#how-should-i-write-my-commits). Commit messages are automatically pulled into your app's changelog for each release and should be clear and understandable to the end user.
-- Add the new app to the release-please-config.json file so that the release workflow will include the new app. The release workflow encapsulates versioning as well as the creation and maintenance of a changelog,
 - Fill out the pull-request template as much as possible. Screenshots, images, and videos are often helpful for providing context and are encouraged.
 - Comment your changes, either in GitHub or the code itself to provide reviewers more context than they might get otherwise.
 - Ensure your changes are wholly contained within your app's folder, and do not make alterations to other folders or the root level. If you have need for root-level changes, please create an issue instead.
 - Prevent / reduce references to external resources and code. The `marketplace-partner-apps` is intended to be open and transparent and any external resources will be _highly_ scrutinized.
```

Comment:
Can we add one more to the list here? 
"Submit the Contentful Marketplace Listing Questionnaire: https://contentful.typeform.com/to/fHkaviJq if you haven't already"